packages/studio-ui/src/AppWithLazyLoading.tsx

```diff
@@ -18,6 +18,7 @@ export default function AppWithLazyLoading() {
 
   useEffect(() => {
     loadComponents();
+    void loadStyling();
```

Comment:
do we need a `stylingLoaded` check like we have `componentsLoaded`? otherwise I imagine the Studio app could be rendered before the styling finishes loading?

Reply:
Will address in a future PR!